play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -42,275 +42,265 @@ Acquired: . . . Who Needs Nano-Technologies?
-this feat.
+Prerequisites: INT 3
+As a free action (in or out of combat) you can use any keyboard, plug into a 
+terminal, or use a monitor to perform a computer inquiry or operation. 
++15% to ONE pertinent skill check (hacking, etc.) while using this feat,
+to be chosen at the time of taking this feat.
 
 First Shot Counts: 
-Prerequisites:
-When reloading, if you Critically succeed, the first round out of that 
-magazine deals Extra-Crit damage.
+Prerequisites: LCK 6
+When reloading, if your reload roll is higher than an 80 (including any applicable skills) or
+if you roll inside your crit range, your first shot is an automatic critical + 10 bonus damage,
+for a total of 30 bonus damage and other possible critical effects. (Instead, if you have Double Tap, 
+you get +10 bonus damage for each Double Tap bullet).
+You have to reload a fresh magazine in order to trigger the bonus again. In other words, you cannot use
+First Shot Counts more times in one encounter than the number of fresh magazines you are carrying. If
+your weapon damages an area, First Shot Counts does not take effect.
 
 Focus Energy: 
 Prerequisites:
-Spend 2 Actions to increase the effects of a spell or Nanite based 
-action by 20%, (20% better roll, 20 damage, up to you.)
+Spend 1 Action to increase the effects of a spell or Nanite based 
+action by 30%, (30% better roll, +30% damage [drop the remainder], up to you.)
 
 Four on the Floor:
 Prerequisites: Pilot or 20 skill in piloting
-Tires were meant to be balded. You get a +20 to any difficult maneuver with a 
+Tires were meant to be balded. You get a +25 to any difficult maneuver with a 
 land vehicle in which you do not catch air. This works with vehicles that have 
 treads or some other method of land support as well.
 
-Gunsmith: 
+Novice Gunsmith: 
 Prerequisites:
 Combine up to 2 weapon total conversions. Ask GM about applicability of 
 desired conversations.
 
 Hands of an artist:
-Prerequisites: Dex 7 or higher
+Prerequisites: DEX 7
 You use your DEX modifier instead of your PER modifier for rolls to hit with 
 ranged weapons. In taking this feat you sacrifice the ability to use PER on 
 rolls dealing with using a weapon.
 
-Improvise Weapon:
-Prerequisites: Unarmed > 20
-You do not take a penalty when using an improvised weapon. Chair legs can be 
-used as bats, fan blades as sowrds, etc., etc.
+POTENTIALLY REMOVE
+	Improvise Weapon:
+	Prerequisites: Skill [Melee Combat] at 20
+	You do not take a penalty when using an improvised weapon. Chair legs can be 
+	used as bats, fan blades as swords, etc., etc.
 
 Last Shot Counts: 
-Prerequisites:
-The last shot in every mag deals Extra-Crit damage, but your reload DC is 
-twice as high.
+Prerequisites: PER 5
+The last shot in every mag deals is an automatic critical + 10 bonus damage,
+for a total of 30 bonus damage. If your weapon has a magazine size of 1 or damages an area,
+Last Shot Counts does not take effect.

```

Comment:
Needs similar balances to First Shot Counts?

Reply:
I think Taylor thought it was fine to allow _first shot counts_ to occur. I think at least one of these should be allowed, and it made more sense to allow first shot counts since it isn't necessarily an absolute that you will get bonus damage on your first shot. 
In other words, having both _last shot counts_ and _first shot counts_ to proc on single fire is a whole lot of damage, and in some ways, _first shot counts_ is the weaker feat (since it has a chance to fail) if we were going to allow one of them to occur.